firesql/sql/sql_objects.py

```diff
@@ -51,6 +51,7 @@ class SQL_ColumnRef():
   table: str
   column: str
   func: str
+  alias: str = None
```

Comment:
Strangely, this was previously missing, thereby disabling officially supporting column aliasing.  This is part of the solution to enable that.